app/controllers/orders_controller.rb

```diff
@@ -21,36 +21,32 @@ def create
       redirect_to order_path(@order)
       flash[:notice] = 'Order posted successfully'
     else
+      flash[:alert] = 'Order not created'
       render :new
       # render 'support/orders/show'
-      flash[:alert] = 'Order not created'
```

Comment:
changed order, flash messages should come before the render and redirect

Need to fix the above `flash[:notice]`